### DISCUSSION TITLE
learning/

### DISCUSSION THREAD

**Discussion by giscus** [dragomano/alpinejs-russian#14] [Q&A]
# learning/

Список полезных русскоязычных ресурсов о фреймворке Alpine.js

https://alpinejs.dragomano.ru/learning/

**Comment by wtwcoder** [comment]
Огромное спасибо за сайт! Очень качественная подача.
Нашел вот такую библиотеку, было бы здорово по ней тоже туториал сделать - https://alpinejs.dragomano.ru/learning/

**Reply by wtwcoder** [reply]
Ошибся, ссылка вот - https://github.com/marcreichel/alpine-auto-animate

**Reply by dragomano** [reply]
Она давно не обновлялась, вы проверяли работоспособность с 3 версией Alpine?

**Reply by dragomano** [reply]
Проверил сам, в принципе работает: https://alpinejs.dragomano.ru/plugins/auto-animate/
